requestResponseHandler.ts

```diff
@@ -48,9 +51,39 @@ export default class QueueITRequestResponseHandler {
             }
             this.httpContextProvider = new CloudflareHttpContextProvider(request, bodyText);
 
+            let decodedIdToken;
+
             const queueitToken = getQueueItToken(request, this.httpContextProvider);
+            const idToken = getIdToken(request, this.httpContextProvider);
             const requestUrl = request.url;
 
+            if (idToken) {
+                decodedIdToken = jwt.decode({token: idToken, secret: AUTH_SECRET });
+
+                const flags = await getFlagsForUser({
+                    key: decodedIdToken["sub"],
+                    email: decodedIdToken["email"],
+                    phoneNumber: decodedIdToken["phone_number"],
+                    ip: request?.ip,
+                });
+
+                const editionQueueEnabledVariationValue = await getVariationValueForFlag({
+                    flag: 'isEditionQueueEnabled',
+                    key: decodedIdToken["sub"],
+                    email: decodedIdToken["email"],
+                    phoneNumber: decodedIdToken["phone_number"],
+                    ip: request?.ip,
+                });
+
+                const variationValue = JSON.parse(editionQueueEnabledVariationValue);
+                const editionFlowId = request.payload.variables.input.editionFlowId;
```

Comment:
could we handle a case where editionFlowId is empty or null? in case we forgot to set it in launch darkly

Reply:
Yeah for sure. That makes sense

Reply:
@calvin-genies any update on this?